js/gameFunctionality.js

```diff
@@ -1,219 +0,0 @@
-  fill(255);
-  textAlign(CENTER);
-
-  if (startScreenCounter > 10) {
-    // startScreenCounter starts when "orb" appeared
-    textSize(75);
-    fill(black, alphaCount * 2); // using alphaCount to have a fading in
-    text("orb", width / 2, height / 2 + 25);
-  }
-
-  if (startScreenCounter > 50) {
-    textSize(40);
-    fill(white);
-    text("Press 'orb' to play", width / 2, height / 2 + 300);
-  }
-  pop();
-  // Growing/shrinking cycle
-  if (circleD > 200) {
-    grow = false;
-  }
-  if (circleD < 181) {
-    grow = true;
-  }
-  // different tempo at smallest and biggest size
-  if (grow == true) {
-    if (circleD > 197) {
-      circleD += growAmount / 4;
-      startScreenCounter++;
-    } else if (circleD < 180) {
-      circleD += growAmount;
-    } else {
-      circleD += growAmount / 2;
-      if (startScreenCounter > 0) alphaCount += 2; // "orb" fades in, after some time
-    }
-  } else {
-    if (circleD < 183) {
-      circleD -= growAmount / 4;
-    } else {
-      circleD -= growAmount / 3;
-    }
-  }
-}
-
-function gameOverScreen() {
-  push();
-  textFont(font);
-  background(255);
-  fill(black);
-  noStroke();
-  circle(width / 2, height / 2, gameOverCircleD);
-
-  if (gameOverCircleD > 200) {
-    gameOverCircleD -= 10;
-  }
-
-  if (gameOverCircleD < 201) {
-    textSize(40);
-    fill(white);
-    textAlign(CENTER);
-    fill(white, GameOverAlphaCount);
-    text("Game", width / 2, height / 2 - 25);
-    text("Over", width / 2, height / 2 + 15);
-    fill(black);
-    text(`Your Score is ${score}`, width / 2, height / 2 + 150);
-
-    if (GameOverAlphaCount < -600) {
-      textSize(75);
-      fill(white);
-      text("orb", width / 2, height / 2 - 15);
-      if (gameOverToggle1) {
-        infobop.play();
-        gameOverToggle1 = false;
-      }
-    }
-
-    if (GameOverAlphaCount < -1600) {
-      if (gameOverToggle2) {
-        infobop.play();
-        gameOverToggle2 = false;
-      }
-      textSize(40);
-      fill(black);
-      text("Press 'orb' to play again", width / 2, height / 2 + 300);
-      textSize(17);
-      text("Which is not important, because this", width / 2, height / 2 + 190);
-      text("is just a silly game on the internet ♡", width / 2, height / 2 + 212);
-    }
-  }
-  pop();
-  if (gameOverCircleD > 199) {
-    grow = false;
-  }
-  if (gameOverCircleD < 181) {
-    grow = true;
-  }
-
-  if (grow == true) {
-    if (gameOverCircleD > 197) {
-      gameOverCircleD += growAmount / 4;
-    } else {
-      gameOverCircleD += growAmount / 2;
-      GameOverAlphaCount -= 25;
-    }
-  } else {
-    if (gameOverCircleD < 183) {
-      gameOverCircleD -= growAmount / 4;
-    } else {
-      gameOverCircleD -= growAmount / 3;
-    }
-  }
-}
-// Checks for mouse clicks during start and game over screen
-function mousePressed() {
-  if (startScreenShow && mouseX > width / 2 - 100 && mouseX < width / 2 + 100 && mouseY > height / 2 - 100 && mouseY < height / 2 + 100) {
```

Comment:
yikes. if statements like this are incomprehensible "at a glance". And I certainly don't want to look at the whole thing and think about it to understand your code. 
```js 
const xIsTheCase =  superlong && completely && unreadable && boolean || logic
if (xIsTheCase)
```